app/main.py

```diff
@@ -2,0 +1,6 @@
+def copy_file(command: str) -> None:
+    com_list = command.split()
+    if len(com_list) != 3 or com_list[0] != "cp" or com_list[1] == com_list[2]:
+        return
+    with open(com_list[1], "r") as file_in, open(com_list[2], "w") as file_out:
```

Comment:
It's a good practice to handle potential exceptions when dealing with file operations. Consider adding error handling for cases where the file might not exist or there are permission issues.